.github/workflows/archived/enhanced-ci-cd.yml

```
@@ -121,7 +121,7 @@ jobs:
       - name: Run integration tests
         env:
           ConnectionStrings__DefaultConnection: "Host=localhost;Database=terrafusion_test;Username=postgres;Password=test_password"
-          ConnectionStrings__Redis: "localhost:6379"
+          ConnectionStrings__Redis: "localhost:\${{TF_REDIS_PORT:-6379}}"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 124: found unknown escape character

(syntax-check)